Marlin/pins_RIGIDBOARD_V2.h

```diff
@@ -24,6 +24,8 @@
  * RIGIDBOARD V2 Arduino Mega with RAMPS v1.4 pin assignments
  */
 
+// DAC_STEPPER_DFLT moved to Configuration_adv.h in March 2017
```

Comment:
The `DAC_STEPPER_DFLT` setting is recent, and never meant to be a user-facing setting, which is why it has an uglier-than-usual name. So I recommend also renaming this setting in addition to moving it to `Configuration_adv.h`. Either use the suggested name, or just make it more readable, as in `DAC_STEPPER_DEFAULT`. Whatever seems more forward-thinking, I guess….